### PR TITLE
[T3A2][F11-A4]ChiangWengKiat

### PR DESCRIPTION
**Pull request by wengkiat**
Encapsulated feedbackToUser and created a getter method for it

### CODE REVIEW

**Comment by wengkiat** [comment]
Ready for Review

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -32,5 +32,8 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
-
+    /** Returns feedback message to be shown to user */
+    public String getFeedbackToUser(){
```
Comment (cmkumar87) [inline src/seedu/addressbook/commands/CommandResult.java]: consider removing useless header comments inserted by auto-gen. 
Accessors usually don't need header comments as their names are self-explanatory.

**Comment by cmkumar87** [comment]
@wengkiat Please acknowledge and close PR. We are now advised to follow this protocol for PR closing to ensure student read our comments.

**Comment by wengkiat** [comment]
@cmkumar87  read then feedback, thank you!

**Pull request closed without merging**